src/examples/wordpress-site.yml

```
@@ -53,6 +53,7 @@ usage:
                       requires:
                           - wpengine/lint
                           - wpengine/codeception
+                          - wpengine/backup_prod
```

```suggestion
                          - wpengine/backup
```

The job name is just `backup`, right?

---

Fixed.